app/src/main/java/com/imdbmovieapp/data/local/dao/MovieDao.kt

```diff
@@ -0,0 +1,22 @@
+package com.imdbmovieapp.data.local.dao
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.imdbmovieapp.data.local.entity.MovieEntity
+
+@Dao
+interface MovieDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insert(movieEntity: MovieEntity)
```

Comment:
please rename functions to self-descriptive names. insert ( insert what ? ).
You can try insertMovie, or something like this